mne/inverse_sparse/tests/test_gamma_map.py

```diff
@@ -130,12 +130,6 @@ def test_gamma_map_vol_sphere():
                                     eeg=False, meg=True)
 
     alpha = 0.5
-    pytest.raises(ValueError, gamma_map, evoked, fwd, cov, alpha,
-                  loose=0, return_residual=False)
-
-    pytest.raises(ValueError, gamma_map, evoked, fwd, cov, alpha,
-                  loose=0.2, return_residual=False)
-
```

Comment:
rather than remove, I'd leave one of them (maybe loose=0.2) to actually run as a smoke test, assuming it doesn't add a ton of time

Reply:
Good idea. Test times don't seem too bad.